README.md

```diff
@@ -91,6 +91,24 @@ These environment variables can be set during container invocation as well. This
 docker run -d -p 80:9080 -p 443:9443 -e WLP_LOGGING_CONSOLE_FORMAT=JSON -e WLP_LOGGING_CONSOLE_LOGLEVEL=info -e WLP_LOGGING_CONSOLE_SOURCE=message,trace,accessLog,ffdc,audit open-liberty:latest
 ```
 
+Disabling messages.log / trace.log output can be achieved by creating a `bootstrap.properties` file with the following environment variables.
+```
+com.ibm.ws.logging.trace.file.name=stdout
+com.ibm.ws.logging.trace.specification=com.ibm.ws.*=fine
+
+com.ibm.ws.logging.message.format=json
+com.ibm.ws.logging.message.source=
+
+com.ibm.ws.logging.console.source=message,trace
+com.ibm.ws.logging.console.format=json
+```
+
+Be sure to include the newly generated file into your Dockerfile as shown in the snippet below. 
+
+```dockerfile
+COPY --chown=1001:0  bootstrap.properties /config/
+```
+
```

Comment:
I think we need to combine this example with the one above so that we don't make the logging section too long.  See the part that starts with "Configure your Open Liberty docker image to emit JSON formatted logs".  Also, please change the list of sources to include all sources and remove the trace specification line (as we don't want people copying that verbatim).

Reply:
I tried combining both of the examples by allowing the reader to go in 2 different directions with the Dockerfile. Also, I have added the missing sources for the "console" variable ( but kept the sources for "messages" as empty ) and removed the trace specification.

Reply:
We should try to keep this fairly short and leave exploring the other options to the docs.  Given all settings can be set using the bootstrap.properties, I'd suggest we show those (note I added one more), perhaps with some comments embedded
```
# direct events to console in json format
com.ibm.ws.logging.console.log.level=info
com.ibm.ws.logging.console.format=json
com.ibm.ws.logging.console.source=message,trace,accessLog,ffdc,audit

# disable writing to messages.log by not including any sources (optional)
com.ibm.ws.logging.message.format=json
com.ibm.ws.logging.message.source=

# disable writing to trace.log by only sending trace data to console (optional)
com.ibm.ws.logging.trace.file.name=stdout
```

Reply:
we can also say that many can also be set using env vars and show the docker command line example.

Reply:
Makes sense, I have changed the logging section to focus on the bootstrap.properties file this time with the above snippet.